release-team/role-handbooks/release-signal/README.md

```diff
@@ -0,0 +1,563 @@
+- Experience with quality assurance and bug-tracking systems.
+- Experience with the GitHub automation.
+
+### Additional Requirements for Leads
+
+In addition to the above requirements for shadows, most of which become prerequisites, Release Signal Leads must:
+
+- Have the ability to add a milestone to issues, so must be a member of
+  the [milestone maintainers](https://github.com/orgs/kubernetes/teams/milestone-maintainers). Reach out to the Release Add for the same.
+- Have a working knowledge of our various test infrastructure tools, such as [Testgrid](https://github.com/GoogleCloudPlatform/testgrid), [Triage](https://github.com/kubernetes/test-infra/tree/master/triage), [Spyglass](https://docs.prow.k8s.io/docs/spyglass/), [Prow](https://docs.prow.k8s.io/docs/), and [Tide](https://docs.prow.k8s.io/docs/components/core/tide/).
+- Have a working knowledge of GitHub labelling (using [prow](https://prow.k8s.io/command-help)) and labels (main: SIG, Area, Priority, Kind, Triage) used by the project. Release Signal leads must be able to identify
+  relevant issues and PRs for a given milestone.
+- Understand what defines a Release Blocking issue or PR, or know who to contact to determine that information.
+- Release Signal lead needs to understand what tests matter and how our testing infra is wired together.
+    - They can ask previous Release Signal leads for advice.
+    - They can ask SIG-Testing for guidance.
+- The Release Signal lead should take
+  the [Inclusive Speaker Orientation (LFC101)](https://training.linuxfoundation.org/training/inclusive-speaker-orientation/)
+  training course.
+
+The core responsibility of the Release Signal lead is to foster a culture of continuous test integration, fail-fast and fix-fast
+mindset and strive for continuously passing tests that provide an accurate positive signal to the release team to ensure that
+releases go out on time with high quality. It is critical to maintain a sustained focus on test health through the entire release
+cycle, as opposed to accepting destabilizing test failures during active enhancement development followed by an extended test
+stabilization phase.
+
+## Timeline
+
+### Onboarding
+
+- Update
+  the [ci signal team in the teams.yaml](https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-release/teams.yaml)
+  which grants access to the [ci signal project board](https://github.com/orgs/kubernetes/projects/68/). _Coordinate with the
+  release lead to make multiple changes to this file in one PR._
+- Adding the lead and (more experienced) shadows as milestone maintainers in
+  the [teams.yaml](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-release/teams.yaml).
+  _Coordinate with the release lead to make multiple changes to this file in one PR._
+- Plan release support and status reporting.
+  See [CI Signal GitHub Projects Board, view: 1.XX-reporting](https://github.com/orgs/kubernetes/projects/68). If the view of the
```

Comment:
The same here: how about Bug Triage board?

Reply:
https://github.com/kubernetes/sig-release/pull/2352#discussion_r1397659054

Reply:
Done.